.dockerignore

```diff
@@ -1 +1,5 @@
+.cache
+examples/dig/dig
```

Comment:
This ignores them when building the docker container, ie it doesn't pass (usually darwin) binaries to the (linux) container. They are still built by `make ci` within the docker container.

Short: this has no effect on the build other than making the tarball sent to the docker daemon smaller. https://docs.docker.com/engine/reference/builder/#dockerignore-file